app/src/main/java/org/oppia/android/app/survey/SurveyActivity.kt

```diff
@@ -48,12 +49,10 @@ class SurveyActivity : InjectableAutoLocalizedAppCompatActivity() {
      */
     fun createSurveyActivityIntent(
       context: Context,
-      profileId: ProfileId,
       topicId: String,
       explorationId: String
     ): Intent {
       val params = SurveyActivityParams.newBuilder().apply {
-        this.profileId = profileId
```

Comment:
This has been removed but I do not see a replacement with the decorator here or at the function call site(SurveyWelcomeDialogFragmentPresenter.kt)